### DISCUSSION TITLE
PMmatrixRelTime

### DISCUSSION THREAD

**Discussion by livPKPD** [LAPKB/Pmetrics#79] [Q&A] [answered]
Hi I am using PM version 2.0.0 in R 4.2.0.

I have some code that used to work:
pmTIME<- PMmatrixRelTime(data=data, idCol= "X.ID", dateCol= "DATE", timeCol= "TIME", format= c("d/m/y", "H:M:S"), evidCol= "EVID")

But now I get this error:
Error in dhours(1) : could not find function "dhours"

Can you help? Thanks!

**Comment by Siel** [comment, marked as answer]
It is working on 1.9.7? I found the error
Some time ago this line was changed using the dhours function: https://github.com/LAPKB/Pmetrics/blob/080575415492768e20f9b41024832fde8bfc0b1f/R/PMmatrixRelTime.R#L74

now, dhours is not an R function but (from what I can find) a function provided by the package lubridate.

Could you please install 'lubridate', load it and check if that solves the error?

if that is the case I could add it as a dependency to the package.

**Reply by livPKPD** [reply]
Yes, that works! Thank you

**Comment by Siel** [comment]
I will then introduce that package as a dependency
